notebooks/ultra96v2/Tensil TCU Demo - YoloV4-tiny.ipynb

```diff
@@ -534,42 +423,59 @@
     "                needs_merge.add(tuple(sorted((i, j))))\n",
     "\n",
     "    def merge(needs_merge):\n",
-    "        result = list()\n",
+    "        result = set()\n",
+    "        discarded = set()\n",
     "        for indices in needs_merge:\n",
-    "            result.append(indices[0])\n",
+    "            idx = indices[0]\n",
+    "            if idx not in discarded:\n",
+    "                result.add(indices[0])\n",
+    "            discarded.add(indices[1])\n",
+    "            if indices[1] in result:\n",
+    "                result.remove(indices[1])\n",
     "        return result\n",
     "\n",
-    "    return list(no_needs_merge) + merge(needs_merge)"
+    "    return sorted(list(no_needs_merge) + list(merge(needs_merge)))"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 19,
+   "execution_count": 64,
    "id": "6eb29f54",
    "metadata": {},
    "outputs": [],
    "source": [
-    "box_idx = non_maximum_suppression(b[0])"
+    "box_indices = non_maximum_suppression(boxes_x0y0x1y1[0])"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": 65,
+   "id": "51094b96",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open('/home/xilinx/coco-labels-2014_2017.txt') as f:\n",
```

Comment:
Do we explain how to get these labels anywhere?

Reply:
Yes, it's in a Yolo tutorial.